### ISSUE TITLE
Generated freezed file has undefined class error

### ISSUE THREAD

**Issue by dannyatthaya** [odroe/prisma-dart#96]
When I run the `dart run build_runner build` command, the generated file has errors which is `Undefined class '$PrismaNullCopyWith'`. This happens when you have prisma scheme nullable field (everything works out when you don't have nullable field).

The dependencies that I have is:
```yaml
dependencies:
  orm: ^2.6.0
dev_dependencies:
  build_runner: ^2.3.3
  freezed: ^2.3.2
  json_serializable: ^6.5.4
```

The scheme that I have is:
```prisma
generator client {
  provider = "prisma-client-dart"
}

datasource db {
  provider = "mysql"
  url      = env("DATABASE_URL")
}

model User {
  id        Int      @id @default(autoincrement())
  email     String   @unique
  name      String
  post      Post?    @relation(fields: [postId], references: [id])
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
  postId    Int
}

model Post {
  id        Int      @id @default(autoincrement())
  name      String
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
  User      User[]
}
```

Here is the screenshot of  the errors:
![image](https://user-images.githubusercontent.com/59720628/211456597-1979c794-9553-499f-ad64-f7751ea878ed.png)

**Comment by medz** [comment]
@dannyatthaya There is a temporary fix:

build.yaml
```yaml
targets:
  $default:
    builders:
      freezed:
        options:
          copy_with: false
          equal: false
```

I will fix this problem in the next version as soon as possible.

**Comment by medz** [comment]
@dannyatthaya Fixed in 2.6.1 version
